mpu6050_userspace/mpu6050_get_temp_C_and_F.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+function view_help {
+	echo "-h, --help	help information"
+	echo "This programm for print temperature from mpu6050 in Celsius and Fahrenheit"
+}
+
+function conv_16_bit_hex_dec_signed {
+    res=$(printf "%d" "0x$1")
+    (( res > 32767 )) && (( res -= 65536 ))
+    echo $res
+}
+function get_temperature {
+    H_BYTE_MPU6060=$(i2cget -y 0 0x68 0x41 b)
+    L_BYTE_MPU6060=$(i2cget -y 0 0x68 0x42 b)
+    TWO_BYTE_HEX="${H_BYTE_MPU6060:2}${L_BYTE_MPU6060:2}"
+    TWO_BYTE_DEC_convert=$(conv_16_bit_hex_dec_signed $TWO_BYTE_HEX )
+#    echo "TWO_BYTE_DEC=$TWO_BYTE_DEC_convert"
+    echo "Temp TWO_BYTE_HEX $( echo "scale=3; $TWO_BYTE_DEC_convert/340+36.53" | bc)"
+    echo "Temp Fahrenheit $(echo "scale=3; (($TWO_BYTE_DEC_convert/340+36.53)*9)/5+32" | bc)"
+	LONG_WORD=$(i2cget -y 0 0x68 0x41 w)
+	CONVERT_LONG_WORD=$(((LONG_WORD & 0xFF)<<8 | (LONG_WORD>>8) & 0xFF))
+	(( CONVERT_LONG_WORD > 32767 )) && (( CONVERT_LONG_WORD -= 65536 ))
+	Celsiy=$(bc -l <<< "scale=4;$CONVERT_LONG_WORD/340+36.53")
+	Fahrenheit=$(bc -l <<< "scale=4;$Celsiy*9/5+32")
```

Comment:
What is the reason for different scale for file and for stdout output? Is the 4th digit after point significant?